tests/storage/migration.go

```diff
@@ -95,6 +112,26 @@ var _ = SIGDescribe("Volumes update with migration", decorators.RequiresTwoSched
 			currentKv.ResourceVersion,
 			config.ExpectResourceVersionToBeLessEqualThanConfigVersion,
 			time.Minute)
+		scName, exists := libstorage.GetRWXFileSystemStorageClass()
```

Comment:
So you're using GetRWXFileSystemStorageClass as a promise that you get a CSI storage class?

If that's the case, maybe create a dummy overloaded GetCSIStorageClass and just have it return GetSnapshotStorageClass (snapshots imply CSI) to make it clearer

Reply:
Not only CSI, but also a one who can use WFFC volume binding mode

Reply:
Any storage class can do WFFC, it's just a matter of declaring it on the storage class object